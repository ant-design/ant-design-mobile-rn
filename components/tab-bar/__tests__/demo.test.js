import rnDemoTest from '../../../tests/shared/demoTest';


rnDemoTest('tab-bar');
