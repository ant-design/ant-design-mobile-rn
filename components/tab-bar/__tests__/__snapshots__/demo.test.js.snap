// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/tab-bar/demo/basic.tsx correctly 1`] = `
<RCTTabBar
  barTintColor="#ccc"
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
  tintColor="#33A3F4"
  translucent={true}
  unselectedTintColor="#949494"
>
  <RCTTabBarItem
    icon={
      Object {
        "uri": "alipay",
      }
    }
    onPress={[Function]}
    selected={false}
    selectedIcon={
      Object {
        "uri": "alipay_sel",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    title="Life"
  >
    <View />
  </RCTTabBarItem>
  <RCTTabBarItem
    badge={2}
    icon={
      Object {
        "uri": "koubei",
      }
    }
    onPress={[Function]}
    selected={true}
    selectedIcon={
      Object {
        "uri": "koubei_sel",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    title="Koubei"
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#efeff4",
            "flexDirection": "row",
            "height": 44,
            "paddingLeft": 8,
            "paddingRight": 8,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
            }
          }
        >
          <TextInput
            allowFontScaling={true}
            clearButtonMode="always"
            editable={true}
            onBlur={[Function]}
            onChangeText={[Function]}
            onClear={[Function]}
            onFocus={[Function]}
            onSubmit={[Function]}
            onSubmitEditing={[Function]}
            placeholder="Search"
            prefixCls="am-search"
            style={
              Array [
                Object {
                  "backgroundColor": "#fff",
                  "borderColor": "#dddddd",
                  "borderRadius": 5,
                  "borderWidth": 0.5,
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 14,
                  "height": 28,
                  "paddingBottom": 0,
                  "paddingLeft": 35,
                  "paddingRight": 35,
                  "paddingTop": 0,
                },
                undefined,
              ]
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <Text
          style={
            Array [
              Object {
                "color": "#cccccc",
                "fontFamily": "antoutline",
                "fontSize": 22,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#cccccc",
                "height": 15,
                "left": 16,
                "position": "absolute",
                "top": 14.5,
                "width": 15,
              },
            ]
          }
        >
          
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "height": 28,
              "justifyContent": "center",
            }
          }
        >
          <Text
            onPress={[Function]}
            style={
              Object {
                "color": "#108ee9",
                "fontSize": 16,
                "paddingLeft": 15,
              }
            }
          >
            取消
          </Text>
        </View>
      </View>
      <Text
        style={
          Object {
            "margin": 50,
          }
        }
      >
        Koubei Tab
      </Text>
    </View>
  </RCTTabBarItem>
  <RCTTabBarItem
    icon={
      Object {
        "uri": "friend",
      }
    }
    onPress={[Function]}
    selected={false}
    selectedIcon={
      Object {
        "uri": "friend_sel",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    title="Friend"
  >
    <View />
  </RCTTabBarItem>
  <RCTTabBarItem
    icon={
      Object {
        "uri": "busi",
      }
    }
    onPress={[Function]}
    selected={false}
    selectedIcon={
      Object {
        "uri": "busi_sel",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    title="My"
  >
    <View />
  </RCTTabBarItem>
</RCTTabBar>
`;
